spec.emu

```diff
@@ -143,22 +235,40 @@ contributors: Ujjwal Sharma, Younies Mahmoud
         1. Let _style_ be ? GetOption(_options_, `"style"`, `"string"`, &laquo; `"long"`, `"short"`, `"narrow"`, `"digital"` &raquo;, `"long"`).
```

Comment:
`"narrow"` is definitely not a good option, since it is potentially ambiguous. I used `"long"` because it's the "default" default value: used for `DisplayNames`, `RelativeTimeFormat` and `ListFormat`.